webrtc-stats.html

```diff
@@ -829,11 +916,21 @@
                 </dt>
                 <dd>
                   <p>
-                    Only valid for video.Same definition as <code>corruptedVideoFrames</code> in
+                    Only valid for video. Same definition as <code>corruptedVideoFrames</code> in
                     Section 5 of [[!MEDIA-SOURCE]].
                   </p>
                 </dd>
                 <dt>
+                  <dfn><code>framesLost</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                    Only valid for video. <code>framesLost</code> is the cumulative number of
+                    full frames lost, as defined in Appendix A (i) of [[!RFC7004]].
+                  </p>
+                </dd>
+                <dt>
```

Comment:
The partial lost frames show up as corrupted frames. However, I can add the partial frames lost for applications to make analysis of why the corruption happened.